db/dcrpg/queries.go

```diff
@@ -454,7 +454,7 @@ func InsertVotes(db *sql.DB, dbTxns []*dbtypes.Tx, _ /*txDbIDs*/ []uint64, fTx *
 			}
 
 			err = agendaStmt.QueryRow(val.ID, index, tx.TxID, tx.BlockHeight,
-				tx.BlockTime.T, lockedIn, activated, hardForked).Scan(&rowID)
+				tx.BlockTime, lockedIn, activated, hardForked).Scan(&rowID)
```

Comment:
Instead of passing a `time.Time`, we pass a `dbtypes.TimeDef` thanks to it being a `sql.Valuer` now.